backend/danswer/auth/users.py

```diff
@@ -8,6 +8,8 @@
 from typing import Optional
 from typing import Tuple
 
+from email_validator import EmailNotValidError
```

Comment:
is there a reason why we use `from pydantic import validate_email` below but `from email_validator import validate_email` here?

Reply:
good catch, thanks!